<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsDialog</class>
 <widget class="QDialog" name="SettingsDialog">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AnkiMorph Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QTabWidget" name="tabWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="note_filters_tab">
        <attribute name="title">
         <string>Note Filters</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QTableWidget" name="note_filters_table">
             <column>
              <property name="text">
               <string>Note Type</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Tags</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Field</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Morphemizer</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Read</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Modify</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QPushButton" name="delete_row_button">
               <property name="text">
                <string>Delete Selected Row</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QPushButton" name="add_new_row_button">
               <property name="text">
                <string>Add New Row</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="extra_fields_tab">
        <attribute name="title">
         <string>Extra Fields</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTableWidget" name="extra_fields_table">
             <column>
              <property name="text">
               <string>Note Type</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Focus Morph</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Highlighted</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Difficulty</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="tags_tab">
        <attribute name="title">
         <string>Tags</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_12">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_10">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>One unknown morph</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Multiple Unknown morphs</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>All morphs known</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QLineEdit" name="ready_tag_input"/>
               </item>
               <item>
                <widget class="QLineEdit" name="not_read_tag_input"/>
               </item>
               <item>
                <widget class="QLineEdit" name="known_tag_input"/>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QPushButton" name="restore_tags_defaults_button">
               <property name="text">
                <string>Restore Default Tags Settings</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="parse_tab">
        <attribute name="title">
         <string>Parse</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_13">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_9">
           <item>
            <widget class="QCheckBox" name="parse_ignore_bracket_contents_input">
             <property name="text">
              <string>ignore content in square brackets []</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="parse_ignore_round_bracket_contents_input">
             <property name="text">
              <string>ignore content in round brackets（）</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="parse_ignore_slim_round_bracket_contents_input">
             <property name="text">
              <string>ignore content in slim round brackets ()</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="parse_ignore_names_morphemizer_input">
             <property name="text">
              <string>Ignore names found by the morphemizer</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="parse_ignore_names_textfile_input">
             <property name="text">
              <string>Ignore names found in &quot;names.txt&quot;</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="parse_ignore_suspended_cards_content_input">
             <property name="text">
              <string>Ignore suspended cards</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <item>
            <widget class="QPushButton" name="restore_parse_defaults_button">
             <property name="text">
              <string>Restore Default Parse Settings</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_7">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="skip_tab">
        <attribute name="title">
         <string>Skip</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_20">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_11">
           <item>
            <widget class="QCheckBox" name="skip_only_known_morphs_cards_input">
             <property name="text">
              <string>Skip cards with only known morphs</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="skip_unknown_morph_seen_today_cards_input">
             <property name="text">
              <string>Skip cards that have unknown morphs already seen today</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="skip_show_num_of_skipped_cards_input">
             <property name="text">
              <string>Show &quot;skipped x cards&quot; notifications</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QPushButton" name="restore_skip_defaults_button">
             <property name="text">
              <string>Restore Default Skip Settings</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_12">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="recalc_tab">
        <attribute name="title">
         <string>Recalc</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_18">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_17">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <property name="topMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Min. interval for known morphs:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="recalc_interval_known_input"/>
             </item>
             <item>
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>days</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_13">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_14">
             <property name="topMargin">
              <number>10</number>
             </property>
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Prioritize morphs based on:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="recalc_prioritize_collection_input">
               <property name="text">
                <string>Frequency in card collection</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>17</width>
               <height>37</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_9">
             <item>
              <widget class="QPushButton" name="restore_recalc_defaults_button">
               <property name="text">
                <string>Restore Default Recalc Settings</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_8">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="shortcuts_tab">
        <attribute name="title">
         <string>Shortcuts</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout_19">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_16">
             <item>
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Run recalc</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Open settings</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_8">
               <property name="text">
                <string>Browse ready cards with same unknown morph</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Browse all cards with same unknown morph</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Set card as known and skip</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Learn card now</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>View card morphemes</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_15">
             <item>
              <widget class="QKeySequenceEdit" name="shortcut_recalc_input"/>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="shortcut_settings_input"/>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="shortcut_browse_ready_same_unknown_input"/>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="shortcut_browse_all_same_unknown_input"/>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="shortcut_known_and_skip_input"/>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="shortcut_learn_now_input"/>
             </item>
             <item>
              <widget class="QKeySequenceEdit" name="shortcut_view_morphs_input"/>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="horizontalSpacer_6">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QPushButton" name="restore_shortcut_defaults_button">
             <property name="text">
              <string>Restore Default Shortcut Settings</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_11">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="restore_all_defaults_button">
         <property name="text">
          <string>Restore All Default Settings</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="ankimorphs_version_label">
         <property name="text">
          <string>AnkiMorphs version: x</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cancel_button">
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="save_button">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
